3.processing_CP_features/extract_sc_features.ipynb

```diff
@@ -0,0 +1,2827 @@
+{
```

Comment:
Should this notebook have a related `.py` nbconverted file checked in along with this PR?

Reply:
Yup! I have just done it!